components/googleMap/googleMap.scss

```diff
@@ -0,0 +1,26 @@
+.ui-google-map__custom-marker {
```

Comment:
`.ui-google-map__custom-marker {
...
}

.ui-google-map__custom-marker:before {
...
}

.ui-google-map__custom-marker:after {
...
}`

@asci Do you mean Is it more pure flat structure (copypaste class names) ??

Then why did we start using sass before? :)